OpenRA.Mods.Common/Activities/Repair.cs

```diff
@@ -49,6 +49,13 @@ public override Activity Tick(Actor self)
 
 			if (health.DamageState == DamageState.Undamaged)
 			{
+				if (host.Owner != self.Owner)
```

Comment:
Because afaik, `self.Owner.Stances[self.Owner] == Stance.Ally`, which means we can't filter on ownership.